base/src/main/java/org/mozilla/jss/provider/java/security/JSSSignatureSpi.java

```diff
@@ -268,7 +268,22 @@ public SHA512RSAPSS() {
     }
     public static class MLDSA extends JSSSignatureSpi {
         public MLDSA() {
-            super(SignatureAlgorithm.MLDSA);
+            super(SignatureAlgorithm.MLDSA65);
+        }
+    }
+    public static class MLDSA44 extends JSSSignatureSpi {
+        public MLDSA44() {
+            super(SignatureAlgorithm.MLDSA44);
+        }
+    }
+    public static class MLDSA65 extends JSSSignatureSpi {
+        public MLDSA65() {
+            super(SignatureAlgorithm.MLDSA65);
+        }
+    }
+    public static class MLDSA87 extends JSSSignatureSpi {
+        public MLDSA87() {
+            super(SignatureAlgorithm.MLDSA87);
         }
```

Comment:
@fmarco76 My question above in JSSProvider.java was actually referring to the classes you added here.  I understand your response above, but now my question is, if they are not to be used, why add them?

Reply:
You are right, the reference were not using the correct classes. At runtime there is no difference (I have tested again) since all these are mapped with the same NSS algorithm for the moment. Btw, I have updated the references to use the correct class. If NSS will support different algorithm specification for the signature this can be updated to  work properly and forcing the key.